simpleclient/src/main/java/io/prometheus/client/LabelValueSanitizer.java

```diff
@@ -0,0 +1,5 @@
+package io.prometheus.client;
+
+public interface LabelValueSanitizer {
```

Comment:
As you say this can also be used for other use cases, not just "sanitization". So maybe the interface should have a more generic name covering all use cases, like `LabelValueTransformer`?

Reply:
I agree with @fstab ... `LabelValueTransformer` seems like it could be used more generically.

Reply:
There may be future use cases where the functionality could be used to transform `String` values other than labels, so I feel it would make more sense to rename the interface to `StringTransformer`.